pkg/controllers/common/utils.go

```diff
@@ -118,8 +118,8 @@ func getDefaultSubnetSetByNamespace(client k8sclient.Client, namespace string, r
 }
 
 func NodeIsMaster(node *v1.Node) bool {
-	for k := range node.Labels {
-		if k == LabelK8sMasterRole || k == LabelK8sControlRole {
```

Comment:
My thought is, can we directly extract this closure function here https://github.com/vmware-tanzu/nsx-operator/blob/07f48519c2622633711e1aa182d7d9e54e3ecdf0/test/e2e/framework.go#L174 and reuse it.